### PR TITLE
Cleanup some functions headers and macro using

### PR DESCRIPTION
**Pull request by Dystopian**
**When merged this pull request will:**
- cleanup frisking (removed in #1612);
- cleanup header format;
- cleanup some FUNC macro using.

### CODE REVIEW

File: addons/interaction/RscTitles.hpp [hunk 1]
```diff
@@ -67,7 +67,7 @@ class RscACE_SelectAnItem {
             text = "";
         };
         class itemList: RscListBox {
-            onMouseButtonDblClick = "_this call ACE_Interaction_fnc_onSelectMenuDblClick";
+            onMouseButtonDblClick = QUOTE(_this call DFUNC(onSelectMenuDblClick));
```
Comment (Dystopian) [inline addons/interaction/RscTitles.hpp]: all these calls in this file don't exist

**Comment by commy2** [comment]
Why DFUNC?

**Comment by PabstMirror** [comment]
`FUNC(` isn't safe in configs
with callstack enabled
```
    #define FUNC(var1) {if(ACE_IS_ERRORED) then { ['AUTO','AUTO'] call ACE_DUMPSTACK_FNC; ACE_IS_ERRORED = false; }; ACE_IS_ERRORED = true; ACE_STACK_TRACE set [ACE_STACK_DEPTH, [diag_tickTime, __FILE__, __LINE__, ACE_CURRENT_FUNCTION, 'TRIPLES(ADDON,fnc,var1)', _this]]; ACE_STACK_DEPTH = ACE_STACK_DEPTH + 1; ACE_CURRENT_FUNCTION = 'TRIPLES(ADDON,fnc,var1)'; private _ret = _this call TRIPLES(ADDON,fnc,var1); ACE_STACK_DEPTH = ACE_STACK_DEPTH - 1; ACE_IS_ERRORED = false; _ret;}
```
defining it globablly actually crashes the game on load
```
        ADDON = QUOTE(call FUNC(createStaminaBar));
```
```
File z\ace\addons\advanced_fatigue\CfgEventHandlers.hpp, line 20: '/Extended_DisplayLoad_EventHandlers/RscDisplayMission.ace_advanced_fatigue': Missing ';' at the end of line
Error context hpp", 20, ACE_CURRENT_FUNCTION, 'ace_advanced_fatigue_fnc_createStaminaBar', _this]]; ACE_STACK_DEPTH = ACE_STACK_DEPTH + 1; ACE_CURRENT_FUNCTION = 'ace_advanced_fatigue_fnc_createStaminaBar'; private _ret = _this call ace_advanced_fatigue_fnc_createStaminaBar; ACE_STACK_DEPTH = ACE_STACK_DEPTH - 1; ACE_IS_ERRORED = false; _ret;}";
ErrorMessage: File z\ace\addons\advanced_fatigue\CfgEventHandlers.hpp, line 20: '/Extended_DisplayLoad_EventHandlers/RscDisplayMission.CfgEventHandlers': '.' encountered instead of '='
Application terminated intentionally
```

I don't think many people are using callstack, so it's not a big deal

**Comment by jonpas** [comment]
Classtack log doesn't even exist at the moment. :P

File: addons/interaction/RscTitles.hpp (hunk 1 above)
Comment (PabstMirror) [reply addons/interaction/RscTitles.hpp]: parent `RscACE_SelectAnItem` isn't used anywhere in ace
might be a remnant from agm

**Pull request merged**